dkms.in

```diff
@@ -827,6 +827,7 @@ get_module_verinfo(){
     vals=
     while read -ra vals; do
         [[ ${#vals[@]} -eq 0 ]] && continue
+        [[ ${vals[0]} ]] || continue
```

Comment:
I have replaced that with a check for a non-empty array, not a non-empty first element.